src/components/ADempiere/ActionMenu/Actions.vue

```diff
@@ -178,56 +180,35 @@ export default defineComponent({
       return actionsList.value[0]
     })
 
-    const clickRunAction = (action) => {
-      console.log('clickRunAction', action)
-    }
-
-    const runAction = (action) => {
-      console.log('runAction', action)
-      action.callBack()
+    /**
+     * Run default action with last parameters
+     */
+    function runDefaultAction() {
+      runAction(defaultActionToRun.value)
     } // end runAction
 
     /**
-     * Get element-ui icon from action
+     * Run selected action
+     * @param action
      */
-    const iconAction = ({ type, action }) => {
-      let icon = 'el-icon-setting'
-      if (type === 'dataAction') {
-        switch (action) {
-          case 'setDefaultValues':
-            icon = 'el-icon-circle-plus-outline'
-            break
-          case 'deleteEntity':
-            icon = 'el-icon-delete'
-            break
-          case 'undoModifyData':
-            icon = 'el-icon-refresh-left'
-            break
-          case 'lockRecord':
-            icon = 'el-icon-lock'
-            break
-          case 'unlockRecord':
-            icon = 'el-icon-unlock'
-            break
-          case 'recordAccess':
-            icon = 'el-icon-c-scale-to-original'
-            break
-        }
-      }
-
-      return icon
+    const runAction = (action) => {
+      action.callBack({
```

Comment:
This should be action.runAction function instead callback function, the name should not be callback.

Reply:
is a list of objects, where each action has a name, an icon and a callBack, and the name of the callBack method could be changed, but that same name must be stored in another property, leaving something similar to:
```javascript
const runAction = (action) => {
  action[actionName]({

  })
}
```

Reply:
just change the name to action

Reply:
changes implemented